### PR TITLE
Fix workspace switch workflow

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Fix https://github.com/astronomer/astro-cli/issues/178

### CODE REVIEW

File: auth/auth.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func Login(domain string, oAuthOnly bool) error {
 
 	workspaces := wsResp.Data.GetWorkspaces
 
-	if len(workspaces) == 1 && len(c.Workspace) == 0 {
```
Comment (andriisoldatenko) [inline auth/auth.go]:
```
✗ go run main.go auth login localhost
 CLUSTER                             WORKSPACE
 localhost                           cjsp2qct2000x0838cmbll6t3

 Switched cluster
Default "Andrii Soldatenko's Workspace" (cjsp2qct2000x0838cmbll6t3) workspace found, setting default workspace.
```

**Comment by schnie** [comment]
@andriisoldatenko what happens when you switch a cluster or re-login to a cluster now if they belong to multiple workspaces? Do you think we should show a workspace selection menu?

**Comment by schnie** [comment]
@andriisoldatenko do you think we could have it so we keep track of the workspace you were last using, per cluster and use that when you switch between them? If we can query for fresh data, I'd say we use the last-used workspace, and if it is now missing, them prompt them with a menu / list to choose from.

**Comment by schnie** [comment]
Works great!

**Pull request merged**